modules/storage-expanding-csi-volumes.adoc

```diff
@@ -0,0 +1,24 @@
+// Module included in the following assemblies
+//
+// * storage/expanding-persistent-volumes.adoc
+
+[id="expanding-csi-volumes_{context}"]
+= Expanding CSI volumes
+
+Support for expanding CSI volumes is enabled by default, but it also requires a specific CSI driver to support volume expansion.
+
+:FeatureName: Expanding CSI volumes
+include::modules/technology-preview.adoc[leveloffset=+0]
+
+[IMPORTANT]
+====
+{product-title} does not ship with any CSI drivers. It is recommended
```

Comment:
I'm unsure if this is really a concept or procedure module. If concept, there's a need for additional background info. If procedure, we need to be more specific about the reader's goals, required actions, anticipated outcomes, etc.

Reply:
Good point. I added additional background to make it a more identifiable concept. It's really just a feature that is now enabled but requires a driver. (It wouldn't make sense as a procedure here because the driver is external and we need to direct them to look up their driver's documentation.)

Reply:
I vaguely remember procedures that are really just "go here and get this" in the docs. I'm not sure if that's an invalid move or just a really efficient one. :)

Reply:
Yeah, hard to say. This module's tricky because installing the ext driver is a prereq but then the feature is turned on by default. It's got the markings of a proc but with no concrete steps after the prereq is met, it ends up as a concept.